app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreetings } from '../redux/greetingReducer';
+const Greeting = () => {
+  const message = useSelector((state) => state.messageReducer.message);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchGreetings());
+  }, []);
+  return (
+    <React.Fragment>
+      <h3>Random Message: {message} </h3>
+    </React.Fragment>
```

Comment:
- [ ] A random greeting gotten from the API endpoint is to be displayed every time a user refreshes the page but right now no message is being displayed. Kindly rectify this issue 💯 
- Your site:
![Screenshot 2022-07-05 163217](https://user-images.githubusercontent.com/85113567/177364501-45636224-fcbf-4331-8f97-aa5c3d47b733.png)